static/app/views/dashboards/widgetBuilder/hooks/useWidgetBuilderState.tsx

```diff
@@ -128,6 +129,27 @@ function useWidgetBuilderState(): {
           break;
         case BuilderStateAction.SET_DATASET:
           setDataset(action.payload);
+
+          if (action.payload === WidgetType.ISSUE) {
+            // Issues only support table display type
+            setDisplayType(DisplayType.TABLE);
+          }
+
+          const config = getDatasetConfig(action.payload);
+          setFields(
+            config.defaultWidgetQuery.fields?.map(field => explodeField({field}))
+          );
+          if (displayType === DisplayType.TABLE) {
+            setYAxis([]);
+          } else {
```

Comment:
Oh good point, this would likely be stale if we had to switch it over! I'll check before merging!

Reply:
There was a small bug here, so I added a local variable to store the new display type, if it gets flipped to table then it'll reset the yAxis, otherwise we keep it